### ISSUE TITLE
Orca doesn't announce default value in drop down

### ISSUE THREAD

**Issue by jrieken** [microsoft/vscode#107662]
Testing #107610

* ubuntu20 w/ orca
* navigate into "Editor: Render Whitespace"
* go through the list of options
* :bug: is option is read out expect for the default option

<img width="859" alt="Screenshot 2020-09-29 at 11 04 17" src="https://user-images.githubusercontent.com/1794099/94537815-8c02f300-0243-11eb-95f3-403952e97fd0.png">

**Comment by sandy081** [comment]
Also, In Mac, voice over does not announce that the option is default.
